doc/integrator/https.rst

```diff
@@ -1,7 +1,5 @@
-.. _integrator_https:
```

Comment:
why removing that?

Reply:
This label is not used, so I simply removed it (to avoid warnings - see above)

Reply:
ok